src/platform/stub-setup.ts

```diff
@@ -16,6 +17,17 @@ const smarthomeStub: {
   },
 };
 
-export function injectSmarthomeStubs() {
+// Injects stubs into the global scope, allowing implicit usage for testing
+export function injectSmarthomeStubs(): void {
```

Comment:
should we do this implicitly on the main module import?

Reply:
Yeah, that makes more sense.  Done